.vsts-ci/windows-daily.yml

```diff
@@ -48,8 +48,6 @@ stages:
 
 - stage: TestWin
   displayName: Test for Windows
-  variables:
-  - group: CLR-CAP
```

Comment:
Did you also delete the variable group from the library in Az DevOps?

Reply:
Yes, I did.